projects/components/src/select/select.component.ts

```diff
@@ -90,6 +91,9 @@ export class SelectComponent<V> implements AfterContentInit, OnChanges {
   @Input()
   public highlightSelected: boolean = true;
 
+  @Input()
+  public type?: SelectType;
```

Comment:
Instead of introducing a new input, can the component set the width of the popover content dynamically to match the width of its trigger element? This would be useful at few other location.

Reply:
I agree about not introducing a new input, but I would actually think set the width based on the width of the options provided - why make it wider than the widest option (with some max-width) ?

Reply:
Now popover content of select has min-width equal to width of trigger element. Example how it look not in pagination component.
Previous:
![not-improved-select](https://user-images.githubusercontent.com/76950513/105479051-a9554480-5cac-11eb-99a6-8c1c076e57b4.png)

Current:
![improved-select](https://user-images.githubusercontent.com/76950513/105479065-ace8cb80-5cac-11eb-87d6-f27802091317.png)

If option in popover wider than trigger element:
![long-option](https://user-images.githubusercontent.com/76950513/105479409-179a0700-5cad-11eb-9ee0-362a78dc66a6.png)